config/boards/jetson-nano.conf

```diff
@@ -0,0 +1,11 @@
+# Jetson NANO
+BOARD_NAME="Jetson Nano"
+BOARDFAMILY="jetson-nano"
+BOOTCONFIG="p3450-0000_defconfig"
+KERNEL_TARGET="current"
+FULL_DESKTOP="yes"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="nvidia/tegra210-p3450-0000.dtb"
+SRC_EXTLINUX="yes"
+SRC_CMDLINE="console=ttyS0,115200 console=tty0"
```

Comment:
If we are going to include EXTLINUX functionality, we should do it right.
https://github.com/armbian/build/blob/master/config/boards/orangepizeroplus.conf#L8
https://github.com/armbian/build/blob/master/lib/distributions.sh#L391